packages/rn-tester/js/RNTesterApp.android.js

```diff
@@ -18,7 +18,7 @@ const RNTesterNavigationReducer = require('./utils/RNTesterNavigationReducer');
 const React = require('react');
 const URIActionMap = require('./utils/URIActionMap');
 
-const nativeImageSource = require('../../Libraries/Image/nativeImageSource');
+const nativeImageSource = require('../../../Libraries/Image/nativeImageSource');
```

Comment:
This is a good example of the next step we'll take later to decouple RNTester from React Native. We should be able to set it up so that `react-native` resolves to `'../../../`, so that all of JavaScript references react-native, even if the native dependencies are still relative. Then we would work to treat the native deps the same way.